experiments/storm/src/main/scala/io/gearpump/experiments/storm/producer/StormProducer.scala

```diff
@@ -38,6 +43,9 @@ private[storm] class StormProducer(gearpumpSpout: GearpumpSpout,
 
   override def onStart(startTime: StartTime): Unit = {
     gearpumpSpout.start(startTime)
+    if (gearpumpSpout.ackEnabled) {
```

Comment:
If ack is not enabled, we need to ack every message at the same time when message is emited.
If ack is enabled, we can leave one message unacked.